### DISCUSSION TITLE
Feature Request: Add the ability to exclude all media from the lffi brochure

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#170] [Ideas] [answered]
Could you include the same option for deleting the media as for the _th_ brochure?

- Exclude **all media** from the _lffi_ brochure

**Comment by sircharlo** [comment, marked as answer]
I'll look into adding this for the next release. It would be an additional toggle, "Exclude images from the _lffi_ brochure"

**Reply by sircharlo** [reply]
Implemented in https://github.com/sircharlo/jw-meeting-media-fetcher/commit/6815ad81b62b016497054c94b605ec85b27f35ae

**Reply by e-Feitosa** [reply]
Thank you very much for your time and work on this program, it has been very helpful.

**Reply by sircharlo** [reply]
Released in #173
